examples/tensor_flow/readme.md

```diff
@@ -0,0 +1,252 @@
+
+The other outputs are dictated by the script. In this case its really two types of files -- the script outputs the training model weights into a Tensor Flow `checkpoint` file, and also outputs two `.json` files containing a map of ID -> word and vice versa. These files will be output to the repo matching the pipeline name: `GoT_train`. We'll use these files in the next pipeline.
+
+
+#### Generating:
+
+This pipeline uses the model from the training stage to generate new text. It needs to consume the trained model as input, as well as the ID -> word and word -> ID json maps to output human readable text. To enable this, we specify the `GoT_train` repo as our input.
+
+Notice that we specify the same image as above, just a different entrypoint via the `transform` property of the manifest. In this step, we're outputting the results to a file called `new_script.txt`
+
+
+#### Running:
+
+Now that we have all of the pieces in place, we can run the pipeline. Do so by running:
+
+`make run`
+
+This creates a pipeline from `pipeline.json`, and since a commit exists on the very first input repo `GoT_scripts`, the pipeline runs automatically. To see what's happening, you can run:
+
+`pachctl list-job`
+
+and 
+
+`pachctl list-commit GoT_generate`
+
+The first command will show you the current status of the jobs. In our case, we expect the training pipeline to run for much longer than the generate step.
+
+#### Results:
+
+By default, we've set the size of the model to train to `test`. You can see this on each of the entrypoints specified in the `pipeline.json` file. For this size model, the training pipeline should run for a couple minutes, and the generate step much quicker than that. Let's take a look at the output.
+
+Once `pachctl list-commit GoT_generate` shows a single commit, we can take a look at the output. To do so, you can run:
+
+```
+pachctl get-file GoT_generate {the commit id from the above command} new_script.txt
+```
+
+And you should see some text!
+
+Keep in mind, the model we just trained was very simplistic. Doing the 'test' model suffices as a proof of concept that we can train and use a neural net using Tensor Flow on Pachyderm. That said, the test model is dumb, and the output won't be that readable.
+
+Actually, you can see how 'dumb' the model is. If you [read through the Tensor Flow example](https://www.tensorflow.org/versions/r0.8/tutorials/recurrent/index.html#run-the-code) they describe how 'perplexity' is used to measure how good this model will perform. Let's look at the perplexity of your model.
+
+```
+pachctl list-commit GoT_trian
+pachctl get-file GoT_train {commit from command above} model_results
+```
+
+Remember, this file is the stdout from the python script while its training the model. It outputs the 'perplexity' at standard intervals on the training set, as well as outputs the 'perplexity' for the validation and test sets. You should see a perplexity of about 600 or less.
```

Comment:
Yeah I think that's a good solution. We should silence fuse errors in the job-shim I think now that we have `Transform.Debug` to turn them on if we ever want them. I'll make an issue for that.